flytekit/core/workflow.py

```diff
@@ -477,7 +478,7 @@ def execute(self, **kwargs):
             return get_promise(self.output_bindings[0].binding, intermediate_node_outputs)
         return tuple([get_promise(b.binding, intermediate_node_outputs) for b in self.output_bindings])
 
-    def add_entity(self, entity: PythonAutoContainerTask, **kwargs) -> Node:
+    def add_entity(self, entity: PythonTask, **kwargs) -> Node:
```

Comment:
i would have liked the name to be add_task
add_subwf
add_lp
and in turn all of them just call add_entity